common/tsconfig/package.json

```diff
@@ -1,7 +1,7 @@
 {
-    "name": "common-tsconfig",
-    "version": "4.4.3",
-    "dependencies": {
-        "namseent": "workspace:^1.0.0"
```

Comment:
I removed `namseent` from `common-tsconfig` to put `cli` to root `package.json`.